ray_on_golem/server/models.py

```diff
@@ -92,6 +92,12 @@ class CreateClusterRequestData(ProviderConfigData):
     pass
 
 
+class CreateClusterResponseData(BaseModel):
+    is_cluster_just_created: bool
```

Comment:
the name of this variable is very awkward imo... 

```suggestion
    initial_startup: bool
```

?